module/renderer/src/iosMain/kotlin/org/cru/godtools/shared/renderer/util/LayoutDirection.ios.kt

```diff
@@ -0,0 +1,19 @@
+package org.cru.godtools.shared.renderer.util
+
+import androidx.compose.ui.unit.LayoutDirection
+import io.fluidsonic.locale.Locale
+
+private val RTL_LANGUAGES = setOf(
+    "ar", // Arabic
+    "he", // Hebrew
+    "ckb", // Sorani Kurdish
+    "ps", // Pashto
+    "fa", // Persian
+    "ur", // Urdu
+)
```

Comment:
@levieggertcru is there an API to determine if a locale is RTL or LTR?

otherwise we can just stick with this hardcoded list we will need to update if new RTL languages get added to GodTools